controllers/mattermost/mattermost/mattermost_test.go

```diff
@@ -285,8 +264,31 @@ func TestCheckMattermost(t *testing.T) {
 
 		err = reconciler.Client.Update(context.TODO(), modified)
 		require.NoError(t, err)
+
+		// Update job should be launched, we expect error
+		err = reconciler.checkMattermostDeployment(mm, dbInfo, fileStoreInfo, currentMMStatus, logger)
+		require.Error(t, err)
+
+		// Assert the job was created
+		job := &batchv1.Job{}
+		err = reconciler.Client.Get(context.TODO(), types.NamespacedName{Name: updateJobName, Namespace: mmNamespace}, job)
+		require.NoError(t, err)
+
+		// TODO: assert owner references are set here
```

Comment:
I think we can address this TODO and add this check now, to make sure new functionality is working properly 🙂

Reply:
I will add this. Thanks